metricbeat/module/ceph/cluster_disk/cluster_disk_integration_test.go

```diff
@@ -15,20 +15,31 @@
 // specific language governing permissions and limitations
 // under the License.
 
+// +build integration
+
 package cluster_disk
 
 import (
 	"fmt"
 	"os"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 )
 
 func TestData(t *testing.T) {
-	f := mbtest.NewEventFetcher(t, getConfig())
-	err := mbtest.WriteEvent(f, t)
-	if err != nil {
+	t.Skip("Skipping data generation test")
```

Comment:
Why do you skip it?

Reply:
What a nice timing :smile: https://github.com/elastic/beats/pull/10990#discussion_r262405773 and more info here https://github.com/elastic/beats/pull/10993#discussion_r261627931